### PR TITLE
Release 1.1.84

### PR DESCRIPTION
**Pull request by leslitagordita**

### CODE REVIEW

**Review by alsaihn — APPROVED** [review]

File: src/content/changelog/4-101-0-2021-08-09.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: '4.101.0: 2021-08-09'
+date: 2021-08-09T04:00:00.000Z
+version: 4.101.0
+changelog:
+  - API
+---
+
+### Added
+
+- You can now deactive Payment Methods by accessing the Payment Method Delete ([DELETE /account/payment-methods/{paymentMethodId}](https://www.linode.com/docs/api/account/#payment-method-delete)) endpoint.
```
Comment (thorn3r) [inline src/content/changelog/4-101-0-2021-08-09.md]:
```suggestion
- You can now deactivate Payment Methods by accessing the Payment Method Delete ([DELETE /account/payment-methods/{paymentMethodId}](https://www.linode.com/docs/api/account/#payment-method-delete)) endpoint.
```

File: src/content/changelog/4-101-0-2021-08-09.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- Linode APIv4 now supports autoscaling for Node Pools using the Linode Kubernetes Engine (LKE) endpoints listed below. **Note**: While the autoscaling feature is supported by the Linode APIv4, issuing requests to these endpoints will not yet take any effect on your Node Pools, until the corresponding Linode Kubernetes Engine updates are released. Please refer to the [LKE Changelog](/changelog/linode-kubernetes-engine/) to learn when autoscaling for Node Pools is released.
```
Comment (thorn3r) [inline src/content/changelog/4-101-0-2021-08-09.md]:
```suggestion
- Linode APIv4 now supports adding an autoscaler configuration for Node Pools using the Linode Kubernetes Engine (LKE) endpoints listed below. **Note**: While adding an autoscaler config is supported by the Linode APIv4, issuing requests to these endpoints will not yet take any effect on your Node Pools, until the corresponding Linode Kubernetes Engine updates are released. Please refer to the [LKE Changelog](/changelog/linode-kubernetes-engine/) to learn when autoscaling for Node Pools is released.
```

**Review by thorn3r — CHANGES_REQUESTED** [review]
just leaving a couple suggestions

File: src/content/changelog/4-101-0-2021-08-09.md (hunk 1 above)
Comment (leslitagordita) [reply src/content/changelog/4-101-0-2021-08-09.md]: Updated!

File: src/content/changelog/4-101-0-2021-08-09.md (hunk 2 above)
Comment (leslitagordita) [reply src/content/changelog/4-101-0-2021-08-09.md]: Updated!

**Review by thorn3r — APPROVED** [review]

**Pull request merged**